cmake should check Jupyter while building documentation

While building documentation, I see : 

```
Submodule update
Submodule path '../external/coreneuron': checked out 'd443f9a839d0b2f2a0c5732f59db65400ec27fa3'
Submodule path '../test/rxd/testdata': checked out 'bdf409f5e534543f9dbe3fe83b5547f3ad9cb518'
Build tutorials
/Users/pramod/Downloads/nrn/docs/tutorials
../build_rst.sh: line 21: jupyter: command not found
make[7]: *** [CMakeFiles/prepare_docs] Error 127
```

This is because we don't check if `jupyter` is installed or not.

--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -193,7 +193,9 @@ endif()
 # =============================================================================
 # Build full docs
 # =============================================================================
-if(DOXYGEN_FOUND AND SPHINX_FOUND)
+find_program(JUPYTER_EXE jupyter QUIET)
+
+if(DOXYGEN_FOUND AND SPHINX_FOUND AND JUPYTER_EXE)
   add_custom_target(
           docs
           COMMAND make doxygen

--- a/README.md
+++ b/README.md
@@ -116,6 +116,10 @@ Please refer to [docs/cmake_doc/options.rst](docs/cmake_doc/options.rst) for mor
 
 NEURON now integrates [CoreNEURON library](https://github.com/BlueBrain/CoreNeuron/) for improved simulation performance on modern CPU and GPU architectures. CoreNEURON is designed as a library within the NEURON simulator and can transparently handle all spiking network simulations including gap junction coupling with the fixed time step method. You can find detailed instructions [here](docs/coreneuron/how-to/coreneuron.md) and [here](https://github.com/BlueBrain/CoreNeuron/#installation).
 
+#### Building documentation
+
+See [docs/README.md](docs/README.md) to install dependencies and build documentation.
+
 <a name="build-autotools"></a>
 ### Build using Autotools
 

--- a/README.md
+++ b/README.md
@@ -176,6 +176,8 @@ file.
 
 For more installation information see: [https://neuron.yale.edu/neuron/download/getdevel](https://neuron.yale.edu/neuron/download/getdevel).
 
-## Developer documentation
+## Documentation
 
-Please refer to [docs/README.md](docs/README.md)
+* See documentation section of the [NEURON website](https://neuron.yale.edu/neuron/docs)
+* See [docs/README.md](docs/README.md) for developers documentation
+* See [github.io](http://neuronsimulator.github.io/nrn/) for latest, nightly snapshot

--- a/docs/README.md
+++ b/docs/README.md
@@ -28,7 +28,14 @@ In order to build documentation locally, you need to pip install the [docs_requi
 pip3 install --user -r docs/docs_requirements.txt --upgrade
 ```
 
-Then in your CMake build folder:
+Also, make sure to have Doxygen installed on your system. With all dependencies installed, configure project with
+cmake as described in [README](../README.md). e.g. In your CMake build folder:
+
+```
+cmake .. -DNRN_ENABLE_INTERVIEWS=OFF -DNRN_ENABLE_MPI=OFF -DNRN_ENABLE_RX3D=OFF -DCMAKE_INSTALL_PREFIX=`pwd`/install
+```
+
+Then build docs using command:
 ```
 make docs
 ```  

